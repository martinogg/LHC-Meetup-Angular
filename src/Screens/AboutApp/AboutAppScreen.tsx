import React, {Component} from 'react';
import {Button, Platform, StyleSheet, Text, View} from 'react-native';
import { AppStyles } from '../../AppStyles'

const instructions = Platform.select({
    ios: 'Press Cmd+R to reload,\n' + 'Cmd+D or shake for dev menu',
    android:
    'Double tap R on your keyboard to reload,\n' +
    'Shake or press menu button for dev menu',
});

interface Props {}

export class AboutAppScreen extends Component<Props> {
    public render() {
    return (
        <View style={AppStyles.container}>
        <Text style={styles.welcome}>Welcome to About Screen native!</Text>
        <Text style={styles.instructions}>To get started, edit App.tsx</Text>
        <Text style={styles.instructions}>{instructions}</Text>
        </View>
    );
    }
}

const styles = StyleSheet.create({
    welcome: {
    fontSize: 20,
    textAlign: 'center',
    margin: 10,
    },
    instructions: {
    textAlign: 'center',
    color: '#333333',
    marginBottom: 5,
    },
});
