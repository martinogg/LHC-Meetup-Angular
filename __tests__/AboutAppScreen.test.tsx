import React from 'react';
import 'react-native';

import { AboutAppScreen } from '../src/Screens/AboutApp/AboutAppScreen'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

const createTestProps = (props: Object) => ({
  navigation: {
    navigate: jest.fn(),
    replace: jest.fn(),
    dispatch: jest.fn()
  },
  screenProps: {
    firebaseConnection: ''
  },
  ...props
});

it('should display AboutAppScreen with no errors', () => {

  const props: any = createTestProps({})
  expect(renderer.create(<AboutAppScreen {...props} />)).toMatchSnapshot();
});

test('send Comment button', () => {

  

})