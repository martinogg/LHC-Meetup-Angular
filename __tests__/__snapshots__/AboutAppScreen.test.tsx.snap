// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display FeedbackScreen with no errors 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#2c3e50",
      "flex": 1,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    Welcome to About Screen native!
  </Text>
  <Text
    style={
      Object {
        "color": "#333333",
        "marginBottom": 5,
        "textAlign": "center",
      }
    }
  >
    To get started, edit App.tsx
  </Text>
  <Text
    style={
      Object {
        "color": "#333333",
        "marginBottom": 5,
        "textAlign": "center",
      }
    }
  >
    Press Cmd+R to reload,
Cmd+D or shake for dev menu
  </Text>
</View>
`;
